src/blog/2025/06/flowfuse-forms-easy-data-collection-factory-floor.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "FlowFuse Forms: Easy Data Collection for Your Factory Floor"
+subtitle: Get Shop Floor Data Directly to Machine Settings
+description: Learn how to create and configure forms in FlowFuse Dashboard to collect data efficiently in industrial applications using Node-RED.
+date: 2025-06-09
+authors: ["sumit-shinde"]
+image: /blog/2025/06/images/building-forms-in-flowfuse.png
+keywords: flowfuse form, node-red form, Recipe Management System, Node-RED Recipe Management, FlowFuse Recipe Management System
+tags:
+   - flowfuse
+---
+
+It's often a pain to get important data from the factory floor. Things like updating recipes, doing quality checks, or setting up machines still rely on old methods – manual notes, slow spreadsheets. This can lead to delays, errors, and a lot of wasted time before anyone can actually use the information. It's especially tough when you need quick feedback from an operator or a specific setting to go straight to a machine.
+
+<!--more-->
+
+This post will show you a better way. We'll look at how FlowFuse Forms on the FlowFuse Dashboard can make collecting factory data much simpler. You'll learn how to build useful forms that connect what your team knows directly to your industrial processes. We'll even walk through a example for digitizing machine recipe updates, showing you how to build this solution yourself, step-by-step, which will include everything we learned so far about form.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- **Running FlowFuse Instance:** Make sure you have a FlowFuse instance set up and running. If you don't have an account, check out our [free trial](https://app.flowfuse.com/account/create).
+- **FlowFuse Dashboard:** Ensure you have [FlowFuse Dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) (also known as Node-RED Dashboard 2.0 in the community) installed and properly configured on your instance.
+- **SQLite Contrib Node:** Install the [node-red-node-sqlite](https://flows.nodered.org/node/node-red-node-sqlite) package, which will be used in the practical example.
```

Comment:
I'd put a sentence in here instead to split rhe core need for Node-RED from the 3rd party nodes.

```suggestion
Then you'll need to add two more sets of nodes to your palette:

- **FlowFuse Dashboard:** Ensure you have [FlowFuse Dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) (also known as Node-RED Dashboard 2.0 in the community) installed and properly configured on your instance.
- **SQLite:** Install the [node-red-node-sqlite](https://flows.nodered.org/node/node-red-node-sqlite) package, which will be used in the practical example.
```